adapter/src/disassembly.rs

```diff
@@ -294,6 +296,16 @@ pub fn sbinstr_to_disinstr(instr: &SBInstruction, target: &SBTarget, resolve_sym
         dis_instr.instruction_bytes = Some(bytes_str);
     }
 
+    if let Some( line_entry ) = address.line_entry() {
+      dis_instr.location = Some( Source {
+        name: Some( line_entry.file_spec().filename().display().to_string() ),
```

Comment:
SBFileSpec should be passed through `DebugSession::map_filespec_to_local` to get a path presentable to VSCode

Reply:
Good call, thanks.

Reply:
Done. I had to refactor this to pass a lambda to sbinstr_to_dinstr. Google helped.